tests/30rooms/11leaving.pl

```diff
@@ -0,0 +1,299 @@
+multi_test "Setup a room, and have the first user leave (SPEC-216)",
```

Comment:
I'd like to keep the progress logging that multi_test provides and that currently doesn't seem possible with prepare.